src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -202,13 +195,11 @@ const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
       </>
     );
   };
-
   return (
     <>
-      {shouldShowForm && <RenderForm />}
+      {shouldShowForm && RenderForm()}
```

Comment:
So I had this calling as a JSX component, and It was giving me issues by losing focus after every keystroke in the name input field. Did some research and this was a possible resolution to that issue. I can try and find an alternative resolution if this is not correct.

Reply:
So here is the why : When your onChange event fires, the callback calls setState with the new title value, which gets passed to your text field as a prop. At that point, React renders a new component, which is why you lose focus.

I have found an alternative I have used the "autofocus" handle on the text input field instead of the amends to that component. This seems to be a more plausible option.

Reply:
Agree - I think I prefer the autofocus option. It is a lot clearer so see a JSX component then to have to figure out what that function call does